<launch>
<master auto="start"/>
  <node pkg="move_base" type="move_base" respawn="true" name="move_base" output="screen">
<!--     <rosparam file="$(find ros_study2015_2dnav)/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find ros_study2015_2dnav)/costmap_common_params.yaml" command="load" ns="local_costmap" /> -->
    <rosparam file="$(find ros_study2015_2dnav)/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find ros_study2015_2dnav)/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find ros_study2015_2dnav)/base_local_planner_params.yaml" command="load" />
    <param name="controller_patience" value="3"/>
    <param name="planner_patience" value="5.0"/>
    <param name="recovery_behavior_enabled" value="false"/>
    <param name="clearing_rotation_allowed" value="false"/>
  </node>
</launch>
